src/main/java/hu/dpc/phee/operator/importer/RecordParser.java

```diff
@@ -364,6 +355,28 @@ private void updateTransferTableForBatch(Long workflowInstanceKey, Long complete
             BatchFormatToTransferMapper.updateTransferUsingBatchDetails(transfer, batch);
             transferRepository.save(transfer);
         }
+    }
 
+    private void updateTransferTableWithFailedDuplicateTransaction(Long workflowInstanceKey, String filename) {
```

Comment:
Again same requirement to make failed transaction handling generic.

Reply:
done